modules/zivid/captureassistant.py

```diff
@@ -0,0 +1,85 @@
+"""Contains capture assistant functions and classes."""
+from enum import Enum
+
+import _zivid
+import zivid._settings_converter as _settings_converter
+
+
+class AmbientLightFrequency(Enum):  # pylint: disable=too-few-public-methods
+    """Ensure compatibility with the frequency of the ambient light in the scene."""
+
+    hz50 = _zivid.captureassistant.AmbientLightFrequency.hz50
+    hz60 = _zivid.captureassistant.AmbientLightFrequency.hz60
+    none = _zivid.captureassistant.AmbientLightFrequency.none
+
+    def __str__(self):
+        return str(self.name)
+
+
+class SuggestSettingsParameters:  # pylint: disable=too-few-public-methods
+    """Input to the Capture Assistant algorithm.
+
+    Used to specify a constraint on the total capture time for the settings suggested by the Capture Assistant,
+    and optionally specify the ambient light frequency.
+    The capture time constraint assumes a computer meeting Zivid's recommended minimum compute power.
+
+    """
+
+    def __init__(self, budget, frequency=None):
+        """Initialize SuggestSettingsParameters.
+
+        Args:
+            budget: max capture time
+            frequency: ambient light frequency
```

Comment:
Should we say more here about type? 
- "Budget" seems to accept a number (interpreted as seconds) or datetime.timedelta.
- "Frequency" only accepts members ("versions?") of AmbientLightFrequency enum. This documentation alone does not hint the user about the existence of this enum.

Reply:
Think also that budget -> max_capture_time

Reply:
Have changed doc strings with fixup. 
There I'm more precise that budget (now max_capture_time) expects an instance of datetime.timedelta. Have addressed frequency (now ambient_light_frequency) in similar manner.

The acceptance of floats for budget may be a side-effect (feature?) of pybind11 - not sure if some action is needed at this point.

Reply:
I think it's fine like this. We don't need to officially support using a float for budget.

Reply:
Could `an instance of zivid.capture_assistant.AmbientLightFrequency` be misleading? It implies that it is a class that you would instantiate with a constructor like
```
my_instance = zivid.captureassistant.AmbientLightFrequency()
```
I think it should say:
`a value of the enum zivid.capture_assistant.AmbientLightFrequency`

Reply:
done in fixup